api/db/repositories/orderHelpers/orderHelpers.js

```diff
@@ -0,0 +1,34 @@
+const productRepo = require('@repos/productRepository')
+
+class OrderHelpers {
+    static async addQuantityAndTotalPrice(order, productId, quantity) {
+        const product = await productRepo.getById(productId)
```

Comment:
I think this could be moved out of helper and be done by Domain instead. Then you would pass `product` as a parameter to this function

Reply:
roger